cypress/integration/InstancesTest.ts

```diff
@@ -12,41 +12,52 @@ describe("Instances Test", () => {
-      cy.ouiaId("create-instance", "PF4/ModalContent")
-        .should("be.visible")
-        .within(() => {
-          cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
-          cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
-            "have.text",
-            "Info alert:Your Smart Events instance will be ready for use shortly after creation."
-          );
-          cy.ouiaId("submit", "PF4/Button").click();
-        })
-        .should("not.exist");
-      //TODO: MGDOBR-706 - uncomment code for the verification that the instance was created (currently mocked server does not support it)
-      /*
+      cy.ouiaId("create-instance", "PF4/ModalContent").then(($modal) => {
+        cy.wrap($modal)
+          .should("be.visible")
+          .within(() => {
+            cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
+            cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
+              "have.text",
+              "Info alert:Your Smart Events instance will be ready for use shortly after creation."
+            );
+            cy.ouiaId("submit", "PF4/Button").click();
+          });
+        cy.wrap($modal, { timeout: 20000 }).should("not.exist");
+      });
+
       cy.ouiaId("Instances list table", "PF4/Table")
         .ouiaId(newInstanceName, "PF4/TableRow")
-        .should("be.visible");
-      */
+        .should("be.visible")
+        .within(() => {
+          cy.get("td:first").should("have.text", newInstanceName);
+          cy.get("td:nth-child(2)").then(($state) => {
+            cy.wrap($state).should("have.text", "accepted");
+            cy.wrap($state, { timeout: 30000 }).should("have.text", "ready");
+          });
+        });
     });
 
     it("Cancel", () => {
       const newInstanceName: string = "Canceled instance";
       cy.ouiaId("create-smart-event-instance", "PF4/Button").click();
-      cy.ouiaId("create-instance", "PF4/ModalContent")
-        .within(() => {
-          cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
-          cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
-            "have.text",
-            "Info alert:Your Smart Events instance will be ready for use shortly after creation."
-          );
-          cy.ouiaId("cancel", "PF4/Button").click();
-        })
-        .should("not.exist");
+      cy.ouiaId("create-instance", "PF4/ModalContent").then(($modal) => {
+        cy.wrap($modal)
+          .should("be.visible")
+          .within(() => {
+            cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
+            cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
+              "have.text",
+              "Info alert:Your Smart Events instance will be ready for use shortly after creation."
+            );
+            cy.ouiaId("cancel", "PF4/Button").click();
+          });
+        cy.wrap($modal, { timeout: 7000 }).should("not.exist");
```

Comment:
Same question here. 7 seconds seems maybe too much? In this scenario the modal should close almost immediately.

Reply:
@kelvah the same problem sometimes happens that the test failed when the default timeout was just 4 seconds. I did not want to solve it when the UI is under development.

Reply:
@kelvah what is your suggestion about timeouts? Do we want solve why 4 seconds is not enough?

Reply:
@kelvah I am not sure about performance in GHA. Localy I have no problem with 4 sec.

Reply:
Thanks for the explanation @sjamboro. I tried to run the tests locally too and, like you say, they complete way faster.
I guess the problem is related to GHA having a limited amount of resources to run. So I guess we have to keep the high timeouts.

Reply:
Sorry for jumping into this conversation
It seems that in certain cases, high timeouts are not enough
![image](https://user-images.githubusercontent.com/22591802/169798492-0bb1d31a-49f4-4da1-87f1-05d53b717a79.png)
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/actions/runs/2370475727

I'm able to run IT locally with success (and fast)

How do we want to proceed @sjamboro @kelvah ?